ext_emconf.php

```diff
@@ -27,7 +27,7 @@
     'uploadfolder'     => '1',
     'createDirs'       => 'uploads/tx_dpf',
     'clearCacheOnLoad' => 0,
-    'version'          => '2.0.0',
+    'version'          => '3.0.0',
```

Comment:
INFO: In future we are going to tag versions from this branch as 4.x

Reply:
Version has been changed.